.changelog/release-v15.2.0.md

```diff
@@ -0,0 +1,294 @@
+
+## 2. Bug Fixes
+
+1. **mojaloop/#3474** CL Cache expiration values higher than default causes transfer to fail ([central-ledger/pull/966](https://github.com/mojaloop/central-ledger/pull/966)), closes [mojaloop/#3474](https://github.com/mojaloop/project/issues/3474)
+2. **mojaloop/#3529** CL high latency in transfer processing([central=ledger](https://github.com/mojaloop/central-ledger/pull/981)), closes [mojaloop/#3529](https://github.com/mojaloop/project/issues/3529)
+3. **mojaloop/3580** fix missing toDestination on handling the fspiop source/destination ([central-ledger/pull/982](https://github.com/mojaloop/central-ledger/pull/982)), closes [mojaloop/3580](https://github.com/mojaloop/project/issues/3580)
+4. **mojaloop/2734** Failures in daily cronjob GP tests ([mojaloop-simulator](https://github.com/mojaloop/mojaloop-simulator/pull/146)), closes [mojaloop/2734](https://github.com/mojaloop/project/issues/2734)
+5. **mojaloop/3589** Bulk API Adapter failing due to object-lib-store bug ([bulk-api-adapter/pull/102](https://github.com/mojaloop/bulk-api-adapter/pull/102))
+6. **mojaloop/3579** Provisioning and GP test failing because of Funds In timing problems ([testing-toolkit-test-cases/pull/129](https://github.com/mojaloop/testing-toolkit-test-cases/pull/129))
+
+## 3. Application versions
+
+1. ml-api-adapter: v14.0.0 -> [v14.0.4](https://github.com/mojaloop/ml-api-adapter/releases/tag/v14.0.4) ([Compare](https://github.com/mojaloop/ml-api-adapter/compare/v14.0.0...v14.0.4))
+2. central-ledger: v17.0.3 -> [v17.3.0]((https://github.com/mojaloop/central-ledger/releases/tag/v17.3.0)) ([Compare](https://github.com/mojaloop/central-ledger/compare/v17.0.3...v17.3.0))
+3. account-lookup-service: 14.1.0 -> [v14.2.5](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.5) ([Compare](https://github.com/mojaloop/account-lookup-service/compare/v14.1.0...v14.2.5))
+4. quoting-service: v15.0.2 -> [v15.4.0](https://github.com/mojaloop/quoting-service/releases/tag/v15.4.0) ([Compare](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.4.0))
+5. central-settlement: v15.0.0 -> [v15.0.4](https://github.com/mojaloop/central-settlement/releases/tag/v15.0.4) ([Compare](https://github.com/mojaloop/central-settlement/compare/v15.0.0...v15.0.4))
+6. bulk-api-adapter: v15.0.3 -> [v16.0.1](https://github.com/mojaloop/bulk-api-adapter/releases/tag/v16.0.1) ([Compare](https://github.com/mojaloop/bulk-api-adapter/compare/v15.0.3...v16.0.1))
+7. central-event-processor: [v12.0.0](https://github.com/mojaloop/central-event-processor/releases/tag/v12.0.0) *(Refer to section 5. BREAKING CHANGES)*
+8. email-notifier: [v12.0.0](https://github.com/mojaloop/email-notifier/releases/tag/v12.0.0) *(Refer to section 5. BREAKING CHANGES)*
```

Comment:
Note that we do have a new version of this service that was upgraded. However we have no way of testing it :(